FreeRTOS/Demo/CORTEX_MPS2_QEMU_IAR_GCC/FreeRTOSConfig.h

```diff
@@ -39,6 +39,12 @@
  * See http://www.freertos.org/a00110.html
  *----------------------------------------------------------*/
 
+#define configUSE_TRACE_FACILITY 1
+#define configGENERATE_RUN_TIME_STATS 1
+
+#define portCONFIGURE_TIMER_FOR_RUN_TIME_STATS()
+#define portGET_RUN_TIME_COUNTER_VALUE() 0
```

Comment:
Setting these to blank will make run time stats meaningless - why not just remove any calls that need these macros and disable generating run time stats. I'm not overly familiar with the capabilities of QEMU but if QEMU can emulate a timer, that should be done here.

Reply:
Good point, pushed a fix for that